### PR TITLE
circulating supply addition to faq

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/integrations/faq.md [hunk 1]
```diff
@@ -112,6 +112,16 @@ Yes. Follow the instructions in the [Ledger support for Nyx documentation](https
 ### Where can I find network details such as deployed smart contract addresses? 
 In the [`network defaults`](https://github.com/nymtech/nym/blob/release/{{platform_release_version}}/common/network-defaults/src/mainnet.rs) file. 
 
+### What is Circulating Supply and how to find out the distribution amount?
+
+The Circulating Supply is the number of available `NYM` tokens in the Nyx blockchain system that are often used by exchanges.
```
Comment (mfahampshire) [inline src/integrations/faq.md]: * what are often used by exchanges? this sentence doesn't grammatically make sense. I don't think you need to mention that its often used by exchanges.. people who are looking for the circulating supply will know what it is when looking for it. 
* remove 'system' 
* its not just on the NYX chain is it - there's also the NYM on Ethereum as ERC20s.

File: src/integrations/faq.md [hunk 2]
```diff
@@ -112,6 +112,16 @@ Yes. Follow the instructions in the [Ledger support for Nyx documentation](https
...
+
+The up-to-date Validator API endpoints can be found via the [Swagger Documentation](https://validator.nymtech.net/api/swagger/index.html). There you will be able to find out using the following API calls to retrieve the correct distribution amount and circulating supply within Nym. 
+
+- GET `/circulating-supply`
```
Comment (mfahampshire) [inline src/integrations/faq.md]: these need context - what are these calls?? No need to put `GET` in there but people need to know that these are the functions that can be called to get this information

**Review by mfahampshire — CHANGES_REQUESTED** [review]

File: src/integrations/faq.md [hunk 3]
```diff
@@ -112,6 +112,22 @@ Yes. Follow the instructions in the [Ledger support for Nyx documentation](https
 ### Where can I find network details such as deployed smart contract addresses? 
 In the [`network defaults`](https://github.com/nymtech/nym/blob/release/{{platform_release_version}}/common/network-defaults/src/mainnet.rs) file. 
 
+### What is Circulating Supply and how to find out the distribution amount?
+
+Circulating supply is the total number of available `NYM`. `NYM` is currently present on the IBC-enabled Nyx blockchain, as well as in ERC20 form on Ethereum Mainnet.
+
+The Validator API endpoints can be found via the [Swagger Documentation](https://validator.nymtech.net/api/swagger/index.html). The following endpoints can be called to retrieve the correct distribution amount and circulating supply within Nym: 
+
+Using this API endpoint calls up-to-date information on various token metrics such as `total_supply`, `mixmining_reserved`, `vesting_tokens` and `circulating_supply` available on the market.
```
Comment (mfahampshire) [inline src/integrations/faq.md]: what are these different metrics? explain

File: src/integrations/faq.md [hunk 4]
```diff
@@ -112,6 +112,22 @@ Yes. Follow the instructions in the [Ledger support for Nyx documentation](https
...
+
+`/circulating-supply`
```
Comment (mfahampshire) [inline src/integrations/faq.md]: formatting could be improved here with some bullet points

**Review by mfahampshire — CHANGES_REQUESTED** [review]
see request changes comments

File: src/integrations/faq.md [hunk 5]
```diff
@@ -127,6 +127,37 @@ We use the [Gravity Bridge](https://github.com/Gravity-Bridge) blockchain to bri
...
+- `mixmining_reserved`- The number of Nym tokens that are reserved for the Mixnet miners who help to power the Nym network.
+
+- `vesting_tokens`- The number of Nym tokens that are subject to vesting, meaning they are gradually released over time to certain stakeholders such as the team, advisors, and early investors.
+
+- `circulating_supply`- The number of Nym tokens that are currently in circulation and available to be traded on the open market, which is calculated by subtracting the `mixmining_reserved` and `vesting_tokens` from the `total_supply`.
+
+Using this API endpoint returns the current value of the total supply of Nym tokens:
+
+```
+/circulating-supply/total-supply-value
```
Comment (mfahampshire) [inline src/integrations/faq.md:173]: what are these code elements? why does this one start with a `/` and the one below doesn't?

File: src/integrations/faq.md [hunk 6]
```diff
@@ -127,6 +127,37 @@ We use the [Gravity Bridge](https://github.com/Gravity-Bridge) blockchain to bri
...
 
+### What is Circulating Supply and how to find out the distribution amount?
+
+Circulating supply is the total number of available `NYM`. `NYM` is currently present on the IBC-enabled Nyx blockchain, as well as in ERC20 form on Ethereum Mainnet.
+
+The Validator API endpoints can be found via the [Swagger Documentation](https://validator.nymtech.net/api/swagger/index.html). The following endpoints can be called to retrieve the correct distribution amount and circulating supply within Nym. 
+
+Using this API endpoint returns information about the circulating supply of Nym tokens:
+
+```
```
Comment (mfahampshire) [inline src/integrations/faq.md:138]: I don't understand the formatting here: how do the bullet points relate to this code block?

Comment (mfahampshire) [reply src/integrations/faq.md:138]: Not very clear - are the bullet points fields that are  returned by that querying that single API endpoint?

**Review by mfahampshire — CHANGES_REQUESTED** [review]

File: src/integrations/faq.md [hunk 7]
```diff
@@ -127,6 +127,64 @@ We use the [Gravity Bridge](https://github.com/Gravity-Bridge) blockchain to bri
...
+- `circulating_supply`- The number of Nym tokens that are currently in circulation and available to be traded on the open market, which is calculated by subtracting the `mixmining_reserved` and `vesting_tokens` from the `total_supply`.
+
+Using this API endpoint returns the current value of the total supply of Nym tokens:
+
+```
+/circulating-supply/total-supply-value
+```
+Query Response:
+
+	1000000000.0
```
Comment (mfahampshire) [inline src/integrations/faq.md]: Add a note : what does this number mean? What denomination is it in ?

File: src/integrations/faq.md [hunk 8]
```diff
@@ -127,6 +127,64 @@ We use the [Gravity Bridge](https://github.com/Gravity-Bridge) blockchain to bri
...
+
+Using this API endpoint returns the current value of the circulating supply of Nym tokens:
+
+```
+/circulating-supply/circulating-supply-value
+```
+Query Response:
+
+	368639460.187887
```
Comment (mfahampshire) [inline src/integrations/faq.md:188]: same as comment above

**Review by mfahampshire — CHANGES_REQUESTED** [review]

File: src/integrations/faq.md (hunk 7 above)
Comment (ghost) [reply src/integrations/faq.md]: is this unit in `uNYM`?

File: src/integrations/faq.md (hunk 8 above)
Comment (ghost) [reply src/integrations/faq.md:188]: is this unit in `uNYM`?

**Pull request merged**